content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,214 @@ description: >
+  
+* IPv6 address - 128 bit IP address assigned to pod interfaces, host interfaces, L2 bridges, L3 routers, tunnel end-points and any other network resources requiring IPv6 connectivity. You can configure one or more IPv6 addresses to an interface. An example is `2001:db8:3333:4444:5555:6666:7777:8888`.
+
+* Dual Stack - IPv4 and IPv6 addresses assigned to pod interfaces, host interfaces, L2 bridges, L3 routers, tunnel end-points and any other network resource requiring IPv4 and IPv6 connectivity.
+
+* L2 bridge - a (virtual) [layer 2](https://en.wikipedia.org/wiki/Data_link_layer) bridge.
+
+* Host Network - Network functions residing in user space or the kernel of physical or virtual nodes. 
+
+* Encapsulation - Ability to encapsulate L2 or L3 packets belonging to an `inner network` with an `outer network` header for transport across the `outer network`. This forms a `tunnel` where the encapsulation function is performed at tunnel ingress and de-encapsulation function is performed at tunnel egress. You can think of pods on nodes as the `inner network` and nodes networked together as the `outer network`. [IPIP](https://www.rfc-editor.org/rfc/rfc2003) and [VXLAN](https://www.rfc-editor.org/rfc/rfc7348) are two examples of encapsulation employed in K8s cluster networks.
+
+* [Virtual overlay network](https://book.systemsapproach.org/applications/overlays.html) - Logical or virtual network of tunnels using encapsulation to connect pods running on different nodes. You can imagine the virtual overlay network runs "on top" of the physical underlay network.   
+
+* Physical underlay network - Network resources such as routers or switches that forward packets between nodes. 
+
+* Virtual ethernet link ([veth](https://man7.org/linux/man-pages/man4/veth.4.html)) - Virtual link allowing you to convey packets between namespaces. 
+
+* [Network namespace](https://man7.org/linux/man-pages/man7/network_namespaces.7.html) - Form of isolation where resources share a network stack, memory, processing and so on.
+
+* [CNI](https://www.cni.dev/) - Container network interface providing network configuration, IP address allocation and possibly CNI-specific methods for packet forwarding to enable inter-pod networking. 
+
+
+## Architecture components
+
+The Kubernetes network model introduces a flexible architecture and accompanying components. You can configure a combination of different components to best meet your cluster networking requirements. 
+
+Figure 1 illustrates the Kubernetes network architecture and accompanying components used in the examples below. 
+
+{{< figure src="/docs/images/k8net-net-arch2.drawio.svg" alt="k8s net arch" class="diagram-large" caption="Figure 1. K8s Network Architecture and Components" >}}
+The architecture and components consist of the following:
```

Comment:
Mm ... With my new reader hat on, I might ask "Example of what? Where did those examples come from?" 

All network systems have an architecture. Maybe we call it something else like "K8s Network Model Examples". Diagrams can be simplified/streamlined to keep it simple for the new reader in this first PR with network pictures. Now new readers can dive into the different examples or use-cases.

Reply:
I'll try an analogy. If you're trying to draw up the [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) pattern, you wouldn't mention the implementation-specific details such as [`NSEditorRegistration`](https://developer.apple.com/documentation/appkit/nseditorregistration). You can do MVC without using Apple's ecosystem.

However, you also might want to provide a non-abstract example to help people learn, and that's fine.

When I commented, there were a bunch of implementation specifics such as veth pairs. I want to avoid implying that these are always part of the architecture.